src/veh_interact.cpp

```diff
@@ -694,6 +701,20 @@ void veh_interact::do_install()
         bool can_install = can_install_part();
 
         const std::string action = main_context.handle_input();
+        if (action == "FILTER" ){
+            filter = string_input_popup( _("Set Filter"),
+                                        50, filter,
+                                        _("{f:} to filter by part flag, {pgt} and {plt} to filter by power, combine with ',', otherwise it searches by name"),
```

Comment:
Feeping creaturism. Search by name would be adequate here. Anything more is of marginal benefit vs the bloating of the implementation.